lib/mix/lib/mix/generator.ex

```diff
@@ -77,15 +77,20 @@ defmodule Mix.Generator do
           [from_file: file] ->
             @file file
             File.read!(file)
+
           c when is_binary(c) ->
             @file {__ENV__.file, __ENV__.line + 1}
             c
+
           _ ->
             raise ArgumentError, "expected string or from_file: file"
         end
 
       require EEx
-      EEx.function_from_string :defp, :"#{name}_template", "<% _ = assigns %>" <> contents, [:assigns]
+
+      EEx.function_from_string(:defp, :"#{name}_template", "<% _ = assigns %>" <> contents, [
+        :assigns
+      ])
```

Comment:
Please break `"<% _ = assigns %>" <> contents` out into a variable so the function call fits on one line.

Reply:
Please check it @ericmj first time touching elixir code so i need your guidance.
Thanks!